azurerm/internal/services/managedapplications/managed_application_resource_test.go

```diff
@@ -94,6 +94,21 @@ func TestAccManagedApplication_update(t *testing.T) {
 	})
 }
 
+func TestAccManagedApplication_parameterValues(t *testing.T) {
```

Comment:
could we add a test that goes string -> non-string -> string and checking to make sure the correct property is used?

Reply:
Added tc